pkg/cmd/cli/describe/describer.go

```diff
@@ -534,6 +534,24 @@ func (d *ImageDescriber) Describe(namespace, name string, settings kctl.Describe
 	return describeImage(image, "")
 }
 
+func describeImageSignature(s imageapi.ImageSignature, out *tabwriter.Writer) error {
+	formatString(out, "\tName", s.Name)
+	formatString(out, "\tType", s.Type)
+	if s.IssuedBy == nil {
+		// FIXME: Make this constant
+		formatString(out, "\tStatus", "Unverified")
```

Comment:
I don't like using the negation - generally we try to have status constants that are significantly different from each other.  The constants are what are important - why not simply output the conditions?